pkg/generated/restapi/configure_timestamp_server.go

```diff
@@ -123,6 +124,20 @@ func httpPingOnly() func(http.Handler) http.Handler {
 	return f
 }
 
+// limitRequestBody restricts the maximum size of incoming request bodies based on the configured "max-request-body-size" value.
+// Requests exceeding the limit are terminated with an HTTP 413 error.
+func limitRequestBody(next http.Handler) http.Handler {
```

Comment:
Can we instead pass in maxRequestBodySize read from the flag value rather than hardcode a default value in the code?